x-pack/plugins/canvas/public/state/selectors/workpad.js

```diff
@@ -113,12 +148,21 @@ export function getElements(state, pageId, withAst = true) {
 }
 
 export function getElementById(state, id, pageId) {
+  // do we need to pass a truthy empty array instead of `true`?
   const element = getElements(state, pageId, []).find(el => el.id === id);
   if (element) {
     return appendAst(element);
   }
 }
 
+export function getNodeById(state, id, pageId) {
```

Comment:
`getElementById` is still in use but maybe not for too long, which is why I didn't worry regularizing it for now - the group templates will definitely bring in some new aspects, which will, I think, make elements and groups more and more similar.

Reply:
Yeah, it seemed like you were moving to "nodes" instead of "element" as the terminology, so a node might be an element or a great of elements.

The only code I see that still uses `getElementById` right now is the tests for the selector ;)